.github/workflows/tuist.yml

```diff
@@ -66,7 +66,7 @@ jobs:
     runs-on: macOS-11
     strategy:
       matrix:
-        xcode: ['12.5.1', '13.0'] # keep 12.5.1 to ensure backwards compatibility
+        xcode: ['12.5.1', '13.0', '13.2'] # keep 12.5.1 to ensure backwards compatibility
```

Comment:
The release should be bumped to use 13.2 as well, as the concurrency problem should be fixed now, right @fortmarek ?

If we do that, then you can just keep 13.2 here as well (if instead we can't bump the release for some reason, then we should still remove 13.0 from here)

Reply:
> The release should be bumped to use 13.2 as well, as the concurrency problem should be fixed now, right @fortmarek ?

We should check that by running `./fourier bundle` and then trying to use the artifacts in the Xcode 12.5 version. I agree I'd remove 13.0 either way.